src/css/vocabulary.css

```diff
@@ -1821,6 +1821,34 @@ body > footer .license svg {
 
 /* program-page context */
 
+/* scroll to top element */
+
+.scroll-to-top{
+    position: sticky;
+    bottom: 20px;
+    left: 100%;
+    width: 2.5em;
+    height: 2.5em;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin-top: calc(100vh + 200px);
+    margin-bottom: 10px;
+
+    background-color: var(--vocabulary-brand-color-dark-tomato);
```

Comment:
a better background color might be the --vocabulary-neutral-lighter-grey as that would pair better with the page itself.

Reply:
@possumbilities on implementing this, I realized that this is the same background color of the body of the page. That means that the background would be largely invisible. Hence, one would only be able to differenciate the scroll-to-top button by the icon alone.
Or perhaps you were referring to the color of the icon and not the background color